From 05dd326460042c0f82234b7299a0327529c98d4f Mon Sep 17 00:00:00 2001
From: allgdante <allan.garret@gmail.com>
Date: Thu, 31 Jan 2019 13:36:34 +0000
Subject: [PATCH 7/7] Remove Locale::gettext from parser tests

The package for Alpine doesn't exists and it seems that the tests run ok
without this
---
 parser/tst/gen-dbus.pl   | 1 -
 parser/tst/gen-xtrans.pl | 1 -
 2 files changed, 2 deletions(-)

diff --git a/parser/tst/gen-dbus.pl b/parser/tst/gen-dbus.pl
index 1fe58108..3fa5d0ce 100755
--- a/parser/tst/gen-dbus.pl
+++ b/parser/tst/gen-dbus.pl
@@ -17,7 +17,6 @@
 #
 
 use strict;
-use Locale::gettext;
 use POSIX;
 
 setlocale(LC_MESSAGES, "");
diff --git a/parser/tst/gen-xtrans.pl b/parser/tst/gen-xtrans.pl
index 8cf077f4..64219cef 100755
--- a/parser/tst/gen-xtrans.pl
+++ b/parser/tst/gen-xtrans.pl
@@ -1,7 +1,6 @@
 #!/usr/bin/perl
 
 use strict;
-use Locale::gettext;
 use POSIX;
 
 setlocale(LC_MESSAGES, "");
-- 
2.20.1

